Documentation/tutorial/howto_filter_out_unwanted_apis.md

```diff
@@ -40,19 +40,19 @@ The path of the configuration file could be specified by the following two ways.
 The format of the filter configuration file
 -------------------------------------------
 
-You could specify `apiRules` with a list of `exclude` or `include` rules.
+Specify `apiRules` with a list of `exclude` or `include` rules.
```

Comment:
You may specify `apiRules` with a list of `exclude` or `include` rules.
Or
`apiRules` may be specified with a list of `exclude` or `include` rules.
Or
You could specify `apiRules` with a list of `exclude` or `include` rules. 
Or
Specify `apiRules` with a list of `exclude` or `include` rules.
Or
Specify `apiRules` by providing a list of `exclude` or `include` rules. 

Phrases are more common for bulleted lists.
Full sentences are more formal.